packages/cli/src/lib/configureStorybook.js

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs-extra'
+
+import { getPaths } from '.'
+
+/**
+ * Configure Storybook for the given template by creating a custom preview config
+ */
+export default function configureStorybook({ force }, newStorybookPreview) {
+  const { storybookPreviewConfig } = getPaths().web
+
+  let finalNewStorybookPreview
+  /**
+   *  Check if storybookPreviewConfig already exists.
+   *  Merge both files if it does.
+   *  By removing import react and export decorator from new config
+   *  And put new config inside current config after last import
+   */
+  if (fs.existsSync(storybookPreviewConfig)) {
+    if (force) {
+      fs.unlinkSync(storybookPreviewConfig)
+      finalNewStorybookPreview = newStorybookPreview
+    } else {
+      const CurrentStorybookConfig = fs
```

Comment:
```suggestion
      const currentConfig = fs
```

I prefer to camelCase regular variables and reserve PascalCase for React components. And I think the variable name should be either `currentStorybookPreviewConfig`, `currentPreviewConfig` or just `currentConfig`. Given the context of this file I think `currentConfig` is enough.